collect_app/src/main/java/org/odk/collect/android/activities/FormDownloadList.java

```diff
@@ -341,8 +331,9 @@ private void downloadFormList() {
     @Override
     protected void onSaveInstanceState(Bundle outState) {
         super.onSaveInstanceState(outState);
-        outState.putBoolean(BUNDLE_TOGGLED_KEY, mToggled);
-        outState.putInt(BUNDLE_SELECTED_COUNT, selectedItemCount());
+        // TODO: Is it safe to remove BUNDLE_TOGGLED_KEY ?
```

Comment:
It shouldn't be a problem to remove all references to it. Do you mean that you want to actually remove the key? I'd tend to just leave it.